### DISCUSSION TITLE
Failed loading lazyvim.config.keymaps

### DISCUSSION THREAD

**Discussion by zentochi** [LazyVim/LazyVim#114] [Q&A] [answered]
Hello, I've just installing lazyvim, and after installing I encounter this error. Why it's showing error?. I'm using neovim nightly 0.9.0 on WSL.

![Screenshot 2023-01-24 085230](https://user-images.githubusercontent.com/102146659/214197276-413b318b-82ab-4e53-9d13-d057d7566de1.jpg)

**Comment by folke** [comment]
Can you update LazyVim and report the error you get?
That stacktrace reports an error on a comment line, so that can't be right

**Reply by zentochi** [reply]
still showing error after update lazyvim via lazyvim dashboard
![image](https://user-images.githubusercontent.com/102146659/214242840-226dfe59-4541-4b12-b776-4a72b1d5e68b.png)

**Reply by folke** [reply]
What is the error. I need the current stack trace

**Comment by folke** [comment, marked as answer]
The error is because you are on an old NEovim nightly. Either upgrade your nightly or install the latest stable

**Reply by zentochi** [reply]
finally it work after installing latest nightly nvim! thanks :)
